src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/court/commonplatform/CommonPlatformCourtEventListenerIntTest.kt

```diff
@@ -58,50 +57,6 @@ class CommonPlatformCourtEventListenerIntTest : MessagingMultiNodeTestBase() {
   @Value("\${aws.court-message-bucket-name}")
   lateinit var s3Bucket: String
 
-  @Test
-  fun `FIFO queue and topic remove duplicate messages`() {
-    stubPersonMatchUpsert()
-    stubNoMatchesPersonMatch()
-    val pnc = randomPnc()
-    val defendantId = randomDefendantId()
-    val firstName = randomName()
-    val lastName = randomName()
-    val nationalInsuranceNumber = randomNationalInsuranceNumber()
-    val hearingId = randomHearingId()
-    blitz(30, 6) {
-      publishCommonPlatformMessage(
-        commonPlatformHearing(
-          listOf(
-            CommonPlatformHearingSetup(
-              pnc = pnc,
-              defendantId = defendantId,
-              firstName = firstName,
-              lastName = lastName,
-              cro = "",
-              nationalInsuranceNumber = nationalInsuranceNumber,
-              hearingId = hearingId,
-            ),
-          ),
-        ),
-      )
-    }
-
-    expectNoMessagesOnQueueOrDlq(courtEventsQueue)
-
-    checkTelemetry(
-      CPR_RECORD_CREATED,
-      mapOf("SOURCE_SYSTEM" to "COMMON_PLATFORM", "DEFENDANT_ID" to defendantId),
-    )
-    checkTelemetry(
-      CPR_RECORD_UPDATED,
-      mapOf(
-        "SOURCE_SYSTEM" to "COMMON_PLATFORM",
-        "DEFENDANT_ID" to defendantId,
-      ),
-      0,
-    )
-  }
```

Comment:
This is just proving how FIFO topics work, nothing to do with the application behaviour